src/mode/dropdown-mode.js

```diff
@@ -24,7 +24,7 @@ export default function DropdownMode(input, emit, opts) {
 
 function autoPosition(input, dp) {
   var inputPos = input.getBoundingClientRect();
-  var docEl = document.documentElement;
+  var docEl = document.body.scrollTop !== 0 ? document.body : document.documentElement;
 
```

Comment:
I wonder, is there a reason not to just use `document.body` in all browsers, instead of `document.documentElement`?

Anyway, I'd think that if we need to distinguish between browsers that have documentElement and those that don't this could just look like this:

`var docEl = document.documentElement || document.body;`

Reply:
To give a quick reply on this, not all browsers (like Chrome) put their scroll position on the body, some put it on the document. That's why you need to check if the body or the document has the scrollposition.
`
but thanks for the update, going to take a look!